modules/custom/openy_search/openy_google_search/openy_google_search.module

```diff
@@ -16,7 +16,9 @@ use Drupal\Core\Url;
 function openy_google_search_theme($existing, $type, $theme, $path) {
   return [
     'openy_google_search' => [
-      'variables' => [],
+      'variables' => [
+        'search_page' => '',
```

Comment:
I think `search_page_url` is more logical name for the variable.

Reply:
Fixed